src/components/LoadingScreen/LoadingScreen.js

```diff
@@ -22,7 +22,7 @@ const LoadingScreen = React.memo(function LoadingScreen({
       setVisible(true)
     }
     return () => clearTimeout(timer)
-  }, [progress])
+  }, [progress, visible])
```

Comment:
@bpierre Not sure on this, was it meant to be that `visible` wasn't included to not trigger an effect when it gets changed from inside?

Reply:
No I must have missed it, it seems pretty safe to add as it’s doing checks before setting `visible`.